### PR TITLE
Fixed the instance list operations.

### PR DESCRIPTION
**Pull request by TimSimpsonR**
- Added ComputeServerNotFound exception to distinguish this more alarming case from simply not finding an instance.
- Made load_server method of instance.models create a context.
- Changed how Instance list was created to add more validation.
- Made the details list view work.
- Massaged view data to match exisiting API.

### CODE REVIEW

**Comment by rnirmal** [comment]
+1

File: reddwarf/instance/views.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #    under the License.
 
 
-class InstanceView(object):
+class InstanceLiteView(object):
```
Comment (ed-) [inline reddwarf/instance/views.py]: Same great Instance taste! Half the attributes!

**Comment by ed-** [comment]
+1 pending the Lite/Detail discussion we're having right now

File: reddwarf/instance/views.py (hunk 1 above)
Comment (hub-cap) [inline reddwarf/instance/views.py]: Remove Lite, and make the other Detail

**Comment by hub-cap** [comment]
+1

**Comment by cp16net** [comment]
+1

**Pull request merged**